i18n/request.ts

```diff
@@ -11,8 +11,8 @@ export default getRequestConfig(async ({ requestLocale }) => {
     locale = routing.defaultLocale;
   }
   // Load multiple translation files
-  const mainMessages = (await import(`@/messages/${locale}/en-US.json`)).default;
-  const templateMessages = (await import(`@/messages/${locale}/templates.json`)).default;
+  const mainMessages = (await import(`@/messages/${locale}/global.json`)).default;
```

Comment:
As discussed, changed names of translation files to `global` and `templateBuilder`